scripts/missions/promathia/1_1_The_Rites_of_Life.lua

```diff
@@ -0,0 +1,124 @@
+require('scripts/globals/missions')
+require('scripts/globals/settings')
+require('scripts/globals/zone')
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.COP, xi.mission.id.cop.THE_RITES_OF_LIFE)
+
+mission.reward =
+{
+    keyItem     = xi.ki.MYSTERIOUS_AMULET,
+    nextMission = { xi.mission.log_id.COP, xi.mission.id.cop.BELOW_THE_ARKS },
+}
+
+mission.sections =
+{
+    -- 1. To start this mission, enter Lower Delkfutt's Tower for a cutscene after installing the Chains of Promathia expansion pack.
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return ENABLE_COP == 1 and currentMission < xi.mission.id.cop.THE_RITES_OF_LIFE
+        end,
+
+        [xi.zone.LOWER_DELKFUTTS_TOWER] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    if prevZone == xi.zone.QUFIM_ISLAND then
+                        return 22
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [22] = function(player, csid, option, npc)
+                    return mission:event(36)
+                end,
+
+                [36] = function(player, csid, option, npc)
+                    return mission:event(37)
+                end,
+
+                [37] = function(player, csid, option, npc)
+                    return mission:event(38)
+                end,
+
+                [38] = function(player, csid, option, npc)
+                    return mission:event(39)
+                end,
+
+                [39] = function(player, csid, option, npc)
+                    player:setMissionStatus(xi.mission.log_id.COP, 0)
+                    player:addMission(xi.mission.log_id.COP, xi.mission.id.cop.THE_RITES_OF_LIFE)
+                end,
+            },
+        },
+    },
+
+    -- 2. After the cutscene has finished, enter Upper Jeuno for another cutscene.
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId and missionStatus == 0
+        end,
+
+        [xi.zone.UPPER_JEUNO] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    return 2
+                end,
+            },
+
+            onEventFinish =
+            {
+                [2] = function(player, csid, option, npc)
+                    player:setMissionStatus(xi.mission.log_id.COP, 1)
+                end,
+            },
+        },
+    },
+
+    -- 3. After the second cutscene finishes, speak to Monberaux (G-10, inside the Infirmary) for final cutscene and a Mysterious Amulet.
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId and missionStatus == 1
+        end,
+
+        [xi.zone.UPPER_JEUNO] =
+        {
+            ['Monberaux'] = mission:event(10),
+
```

Comment:
Depending on Monberaux's other interactions, might want to make this a `progressEvent()` as opposed to `event()` to ensure that its triggered.  The other option would be to set it as `mission:event(10):importantOnce()`